frontend/src/app/Settings/Users/CreateUserForm.tsx

```diff
@@ -85,6 +86,7 @@ const CreateUserForm: React.FC<Props> = ({ onClose, onSubmit }) => {
           value={newUser.firstName}
           required
           onChange={onChange}
+          disabled={isUpdating}
```

Comment:
If it was just one button it might make sense to use a `ref` and disable that way but I guess this is better for a case where it's a lot of elements. Something feels strange about this pattern but that's more of a UI design problem.